es6/README.md

```diff
@@ -48,9 +80,200 @@ const MaxAllowed = 7;
+```
+
+Avoid using `const` for local variables (eslint: [`prefer-const`](http://eslint.org/docs/rules/prefer-const)):
+
+```js
+// good
+const generateGreeting = (name=DEFAULT_NAME) => {
+    let formattedNow = (new Date()).toString();
+
+    return `Hi, ${name} on ${formattedNow}`;
+}
+
+// bad (uses `const` for `formattedNow` local variable)
+const generateGreeting = (name=DEFAULT_NAME) => {
+    const formattedNow = (new Date()).toString();
+
+    return `Hi, ${name} on ${formattedNow}`;
+}
+```
+
+There is a pattern in the industry to use `const` if a variable is never going to be reassigned within a block (see eslint [`prefer-const`](http://eslint.org/docs/rules/prefer-const)). However, this is an abuse of `const` as it is intended for variables that are truly constant. The motivation for this practice is that it can help enforce "immutability" since a `const` variable cannot be reassigned. But immutability and `const` bindings are two separate things. For instance, an object declared `const` can still have its properties mutated.
+
+For more on constants, read [_Learning ES6: Block-level scoping with `let` and `const`_](http://www.eventbrite.com/engineering/learning-es6-block-level-scoping-with-let-and-const/).
+
+**[⬆ back to top](#table-of-contents)**
+
+## Strings
+
+### Template literals
+
+When building up a string, use a template literal instead of string concatenation (eslint: [`prefer-template`](http://eslint.org/docs/rules/prefer-template.html)):
+
+```js
+// good
+const generateGreeting = (name=DEFAULT_NAME) => {
+    const formattedNow = (new Date()).toString();
+
+    return `Hi, ${name} on ${formattedNow}`;
+}
+
+// bad (uses string concatenation)
+const generateGreeting = (name=DEFAULT_NAME) => {
+    const formattedNow = (new Date()).toString();
+
+    return 'Hi, ' + name + ' on ' + formattedNow;
+}
+
+// bad (uses array join)
+const generateGreeting = (name=DEFAULT_NAME) => {
+    const formattedNow = (new Date()).toString();
+
+    return ['Hi, ', name, ' on ', formattedNow].join();
+}
+```
+
+When using template literals, tokens should **not** be padded by spaces (eslint: [`template-curly-spacing`](http://eslint.org/docs/rules/template-curly-spacing)):
+
+```js
+// good
+const generateGreeting = (name=DEFAULT_NAME) => {
+    const formattedNow = (new Date()).toString();
+
+    return `Hi, ${name} on ${formattedNow}`;
+}
+
+// bad (has extra padding around the curlies)
+const generateGreeting = (name=DEFAULT_NAME) => {
+    const formattedNow = (new Date()).toString();
+
+    return `Hi, ${ name } on ${ formattedNow }`;
+}
+```
+
+Use template literals instead of escape single quotes (eslint: [`no-useless-escape`](http://eslint.org/docs/rules/no-useless-escape)):
+
+```js
+// good
+let message = `He's the one!`;
+
+// bad (escapes the single quote)
+let message = 'He\'s the one!';
+
+// bad (uses double quotes to avoid quoting)
+let message = "He's the one!";
+```
+
```

Comment:
Maybe add one where you shouldn't use template literals when it's plain text -- instead just use single quotes. If I'm not mistaken, it's already an eslint rule?